### PR TITLE
Add environment Variables to project

### PR DESCRIPTION
**Pull request by KasperFranz**
This PR adds environment variables to the project and moves hardcoded cache values to the .env file.

I have added the current values as default paths

### CODE REVIEW

**Comment by matkoniecz** [comment]
Thanks for PR! It is nice to see someone else interested in this pile of code :)

File: README.md [hunk 1]
```diff
@@ -66,6 +66,9 @@ Code is GNU GPLv3 licensed.
 
 # Configuration
 
+## Environment variables
+Before running the scripts you should create a `.env` variable (this can be copied from the `.env.example`)
```
[the remark this answers is not in the document]
Comment (KasperFranz) [reply README.md:70]: Yes you are right it should say file.

They get pushed to the OS by load_dotenv() which I have added to script and flush which I believe is the two entry points

**Comment by KasperFranz** [comment]
> Thanks for PR! It is nice to see someone else interested in this pile of code :)

Anything that helps move OSM forward :)

**Comment by KasperFranz** [comment]
@matkoniecz  have you had a chance to review this?

**Comment by matkoniecz** [comment]
I have tested it and it works, I looked at it again: and maybe there should be no default values? Maybe ideally also I would configure proper config file?

But this can be merged separately, I guess

**Pull request merged**